libinitializer/Initializer.cpp

```diff
@@ -128,9 +128,20 @@ void Initializer::init(bcos::protocol::NodeArchitectureType _nodeArchType,
     bcos::storage::TransactionalStorageInterface::Ptr consensusStorage = nullptr;
     if (boost::iequals(m_nodeConfig->storageType(), "RocksDB"))
     {
-        storage = StorageInitializer::build(storagePath);
-        schedulerStorage = storage;
-        consensusStorage = StorageInitializer::build(consensusStoragePath);
+        if (true == m_nodeConfig->storageSecurityEnable())
```

Comment:
```suggestion
            storage =
                StorageInitializer::build(storagePath, m_protocolInitializer->dataEncryption());
            schedulerStorage = storage;
            consensusStorage = StorageInitializer::build(
                consensusStoragePath, m_protocolInitializer->dataEncryption());
```

since `m_protocolInitializer->dataEncryption()` is nullptr when storage-security disabled,  unify the logic to the above code.